performance/billing.py

```diff
@@ -22,10 +23,20 @@ def extract_verifications_by_rp_csv_for_date(date_start):
     :return: pandas.Dataframe with non transformed data with matching columns to the original verifications_by_rp
     """
     date_end = fromisoformat(date_start) + timedelta(days=6)
+    file_name = f'verifications_by_rp_{date_start}_{date_end}.csv'
     verifications_by_rp_csv_path = os.path.join(config.VERIFY_DATA_PIPELINE_CONFIG_PATH,
                                                 'data',
                                                 'verifications',
-                                                f'verifications_by_rp_{date_start}_{date_end}.csv')
+                                                file_name)
+
+    if os.path.exists(verifications_by_rp_csv_path):
+        print(f'Using local copy of {file_name}')
```

Comment:
Now sure if now is the right time for this or not, but this feels like `logging.info` rather than `print` to me.

Reply:
Done and updated